### PR TITLE
{bio,math}[intel/2017a] Albacore v2.0.2, numpy v1.13.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2609.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3c177ddcd85c31fe14e45aeb629dd3ec for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2098.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7d46f56b3e7862f7c874711174e4c4ae for a full test report.

File: easybuild/easyconfigs/a/Albacore/Albacore-2.0.2-intel-2017a-Python-3.6.1.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+pyver = '3.6.1'
+pymajmin = ''.join(pyver.split('.')[:2])
+dependencies = [
+    ('Python', pyver),
+    ('numpy', '1.13.1', versionsuffix),
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/Albacore/Albacore-2.0.2-intel-2017a-Python-3.6.1.eb]: that's part of the `Python`?

Comment (boegel) [reply easybuild/easyconfigs/a/Albacore/Albacore-2.0.2-intel-2017a-Python-3.6.1.eb]: The `numpy` version included there is too old, the `.whl` used here requires numpy 1.13.x

**Pull request merged**